pages/api/auth/[...nextauth].js

```diff
@@ -1,92 +1,96 @@
 import NextAuth from 'next-auth'
 import Providers from 'next-auth/providers'
+import Adapters from 'next-auth/adapters'
+import { v4 as uuidv4 } from 'uuid'
+import Cookies from 'cookies'
 
-// For more information on each option (and a full list of options) go to
-// https://next-auth.js.org/configuration/options
-const options = {
-  // https://next-auth.js.org/configuration/providers
-  providers: [
-        Providers.Facebook({
-            clientId: process.env.FACEBOOK_ID,
-            clientSecret: process.env.FACEBOOK_SECRET
-        }),
-        Providers.GitHub({
-            clientId: process.env.GITHUB_CLIENT_ID,
-            clientSecret: process.env.GITHUB_CLIENT_SECRET
-        })
-    ],
-  // Database optional. MySQL, Maria DB, Postgres and MongoDB are supported.
```

Comment:
I find all these comments useful to help future contributors what are all the possible options. Can you make an argument of why you want them deleted?

Reply:
Or is this an artifact of merging branches? Regardless, I would like to get all these comments back into the file, please

Reply:
I think it makes the file hard to read, and I think the things explained in them is easily accessible through the package documentation. But I also get that they might prove to be useful for someone. I'm fine with both. Should I add them back?

Reply:
Maybe add a single comment like `// For more information about each option, see https://next-auth.js.org/configuration/options`.

Reply:
I really would like to keep them there, so yes, please add them all back; thank you